### ISSUE TITLE
Collaborator Summit 2019 - Berlin Edition - 30th and 31st of May

### ISSUE THREAD

**Issue by mcollina** [nodejs/summit#135]
The Vancouver 2018 Collab Summit is just finished, and it's time we start planning for next year first event.

Who is up for helping with the organization?

I would like to propose we do it again in Berlin before [JSConf.eu](https://2019.jsconf.eu/) on the 30th and 31st of May. I would kindly ask @MylesBorins and @hackygolucky to ask the Board if they could fund a venue for the 2-day event.  

cc @nodejs/tsc @nodejs/community-committee @ladyleet

Registration: https://www.eventbrite.com/e/openjs-collaborator-summit-tickets-55985971548
CFP: https://docs.google.com/forms/d/e/1FAIpQLScneYhY4iqm8x2C2AO9EEzJ5smkOKhLqfxEMQcfNvP5rvXfHg/viewform

**Comment by trivikr** [comment]
Hi @mcollina, I attended Vancouver 2018 Collab Summit and had a great time.
I would like to help organize Berlin summit, is there a wiki/notes on responsibilities of the organizers?

**Comment by jasnell** [comment]
One bit of feedback I heard yesterday was that saving all the lightening talks to the end of day 2 so we're prioritizing the working sessions.

**Comment by thefourtheye** [comment]
I also would like to help with organizing.

**Comment by mmarchini** [comment]
I would like to help to organize as well :)

**Comment by benjamingr** [comment]
I would love to help with this and attend!

**Comment by mcollina** [comment]
> I would like to help organize Berlin summit, is there a wiki/notes on responsibilities of the organizers?

Overall, ensuring that the event run smoothly. These includes:

1. Ensure adeguate space is reserved for the summit, number of rooms, "hallway track", and so on.
2. Determine the agenda before hand, and make sure everything is in place. This is typically a time sync and it includes a lot of guessing and back-and-forth to minimize the chance that a collaborator would like to attend two session at the same time as multiple people are involved in multiple discussions. @joyeecheung defined it as "playing Jenga" :D.
3. MC the event, this typically includes a moment for agenda-bashing at the beginning and presenting the closing event.
4. Deal with problems when they arises, including during the summit. This also include broadcasting our contacts for potential CoC violations during the summit.

**Comment by ryzokuken** [comment]
I would love to help out in any way I can.

**Comment by sagirk** [comment]
I would love to attend and make myself useful in any way I can! ❤️

**Comment by mhdawson** [comment]
+1 to limiting the "talks" to the end and making them short (! the length we had for lightning talks this time).  I think its best if most of the time should be for discussions for the WGs/teams or other specific topics.

**Comment by Trott** [comment]
I'd also OK with leaving a very large chunk of the agenda open until the day of, and then bash the agenda for 20 minutes at the start, unconference style. (I'm also OK if the organizers *don't* want to do it that way. Just a suggestion.)

**Comment by mcollina** [comment]
Considering my experience of the last two, my impression is that we need the core of the work already done. Moving 3 slots took me a couple of hours of work to make (almost) everybody happy.

**Comment by WaleedAshraf** [comment]
✋ for all kind of help in organizing next in Berlin.
I think we should be clear how much time a session would take. For example, Governance went for the whole day (and a half) and that wasn't still enough.

**Comment by MylesBorins** [comment]
If moving three slots took hours, that may be a reason to not overly lock
the agenda in advance. Having more flexibility on the day can allow us to
lock in the agenda based on who is there

**Comment by mcollina** [comment]
I would note that without a predetermined draft + the survey that Joyee set
up we would not be able to be as productive.

The overall feedback from Vancouver 2017 and Berlin 2018 has been that more
structure is needed. I think that Vancouver 2018 was a good balance.
If we swift towards a more unconference style, it’s likely that meeting
agendas and presentations will not be prepared.

One problem that I note is that we lacked several members in a lot of
discussions (modules for example). We should be checking in advance if
members will be there when proposing/scheduling a session. Maybe we should
upvote them in advance?

Il giorno mer 17 ott 2018 alle 16:31 Myles Borins <notifications@github.com>
ha scritto:

**Comment by jasnell** [comment]
For Berlin, we will likely have the advantage of two full days for the summit whereas in Vancouver, because of the code and learn we had only 1.5 days. The additional time and a time boxed agenda set in advance will be helpful, for sure. We should try avoiding overlap in sessions except where we know for certain the participants are ok with it

**Comment by rachelnicole** [comment]
Hey friends, how do I stop receiving notifications from this? I stepped down participation in CommComm & unwatched the repos, but I'm still getting emails.

**Comment by Trott** [comment]
One piece of feedback I've received is that there's a sense that decisions get made at the summits and that excludes people who can't attend these things. Travel fund helps, but aside from the expense, traveling to these things is not easy for some people.

I don't know what the solution to that would be, if any, but it's at least something to be *aware* of.

**Comment by Trott** [comment]
(One solution would be to make it easy to stream all the sessions. Even better if people can participate remotely. But that can be a lot of organizational overhead. Would be awesome if we could do it, though.)

**Comment by ChALkeR** [comment]
@Trott Some of those could not be easily streamed, e.g. I believe that governance meeting format didn't allow to stream it - i.e. it was split to several smaller discussion groups for some time, and it used a real-world stickerboard.

But e.g. the format of the security wg meeting allowed streaming it (it was more like a regular conference discussion with everyone participating), and outside input could have been incorporated e.g. from a chat or something like that. For that format, just streaming + chat could have helped. Or streaming + a GitHub issue for discussion of outside participants - that should reduce the organizational overhead and reduce the barrier ever more.

**Comment by mcollina** [comment]
> But e.g. the format of the security wg meeting allowed streaming it (it was more like a regular conference discussion with everyone participating), and outside input could have been incorporated e.g. from a chat or something like that. For that format, just streaming + chat could have helped. Or streaming + a GitHub issue for discussion from outside participants - that should reduce the organizational overhead and reduce the barrier ever more. Something similar to what is done with Zoom meetings + questions from IRC.

Note that enabling remote participation implies having a ~stable agenda some time in advance.

**Comment by mcollina** [comment]
We have got $30K approved by Board to rent a venue. 

@trivikr @thefourtheye @mmarchini @sagirk  @WaleedAshraf 

Are you still ok in helping organizing?

Also @joyeecheung @jorydotcom @hackygolucky, would you like to help?

**Comment by WaleedAshraf** [comment]
@mcollina Yes, ✋

**Comment by mmarchini** [comment]
Yes, still interested :D

**Comment by joyeecheung** [comment]
Yes :)

**Comment by trivikr** [comment]
Yup!

**Comment by jorydotcom** [comment]
@mcollina happy to drive participation/engagement/planning from the jsf side of the table!

**Comment by thefourtheye** [comment]
@mcollina A little late to the party, but yes, I would love to help.

**Comment by sagirk** [comment]
@mcollina Super late, sorry! Yes, still up for it. 👍

**Comment by chowdhurian** [comment]
Dropping a checklist here of planning items from Vancouver 2018 and to a more limited extend Berlin 2018:

- [ ] j
- [ ] 
- [ ] 


Other peeps involved in planning Vancouver 2018 @mcollina @ladyleet @dshaw, please feel free to add anything I miss

**Comment by chowdhurian** [comment]
Dropping a checklist here of the main work items from past Collab Summits:

- [ ] Call for Sessions issue needs posting about 90 days ahead (see #100 as example), not too early that the topics are less relevant by the time of the Summit, but not too late that people don't have time to prepare
- [ ] [The Session Facilitator Guide](https://github.com/nodejs/summit/blob/master/SESSION_FACILITATOR_GUIDE.md) should be promoted well ahead of time and included in the Call for Sessions. It can be updated on an ongoing basis with past learnings, and new ideas like those discussed above in this thread etc.
- [ ] Maintain a live agenda in the document being PR'd #139

Please feel free to add anything I missed! 🙏🏽 Calling out peeps involved in past planning like Vancouver 2018 @mcollina @ladyleet @dshaw. I was going off of memory from those bi-monthly early morning Hangouts calls in the months leading up to Vancouver 😄

**Comment by chowdhurian** [comment]
I can help too.

**Comment by mcollina** [comment]
How about we do a meeting next week at 6pm UTC on Tuesday the 22nd of January?

cc @trivikr @thefourtheye @mmarchini @sagirk @WaleedAshraf @joyeecheung @jorydotcom @hackygolucky @mbj36 @chowdhurian 

I hope that time work  for most people. I think we should plan to kick off the call for sessions asap.

**Comment by ladyleet** [comment]
let me know and i'll add you to the invite.

**Comment by mbj36** [comment]
That works for me @mcollina @ladyleet

**Comment by ladyleet** [comment]
@mbj36 if you send me an email ladyleet@nodejs.org i can go ahead and add you to the invite.

**Comment by trivikr** [comment]
I'll be in an airplane during the first half-an-hour of the meeting (returning from vacation for MLK Day weekend in the US)

I can join at 6:30pm UTC if the meeting is more than 30 mins long.

**Comment by WaleedAshraf** [comment]
@mcollina works for me.

**Comment by chowdhurian** [comment]
[Agenda skeleton](https://github.com/nodejs/summit/blob/master/2019-05-Berlin/agenda.md) is available now.

**Comment by jorydotcom** [comment]
Hey pals,
Wanted to provide a brief recap of a preliminary chat @mcollina @ladyleet @christian-bromann & I had earlier this week, mainly for us (JSF) to learn more about how the summit has been formatted in the past and to share some ideas we had about ways JSF projects could participate/add value. I imagine we'll be sharing this all again at our meeting next week for the broader group, but the general idea so far that Tracy, Christian, Matteo & I liked was to have a 'showcase' area for all the projects, also have a quick "introduce the projects" session right after the welcome, to do a 'call for sessions' to collect/help organize meetings, and to pre-collect unconf session topics.

These are general ideas and of course need to be discussed further, but wanted to share a direction a few of us think could work really well across Node + jsf projects. more ideas are very welcome!

**Comment by antsmartian** [comment]
@ladyleet Not sure, if I can attend. But will the meeting be recorded? Just for others who can't attend :)

**Comment by ladyleet** [comment]
@antsmartian this one will not be recorded, but i will bring this up during the call.

**Comment by ladyleet** [comment]
Is anyone here from Berlin at all?

**Comment by ladyleet** [comment]
Update - we will start broadcasting these meetings for transparency! :)

**Comment by ladyleet** [comment]
You can view the proposed agenda and action items here: https://docs.google.com/spreadsheets/d/1r5am7B7eczC5OpkpZsyoU71LHrWS6fUS1gvDFfR1ifw/edit#gid=1300552757

**Notes:** 
Call for proposals? Needs to be specific that those specific topics need to have the team leads there. (ie if you are proposing a session there needs to be a critical mass of people there)Need to write the criteria down for session proposal- # of people interested in session- team leads or wg members need to be there
**OWNERS:** Manil + Jory + (Tracy H + Matteo + Mohit) | 2/12/2019?

Set up Eventbrite / Tito for RSVP with information ie: - Where are you from- How likely are you able to go- What topics do you want to etc- How did you hear about it?- Which project/team/working group are you part of?
**OWNERS:** Tracy/Kaelyn | 2/5

Venue 
**OWNERS:** Tracy/Kaelyn/Christian | 2/12/2019?

Merger discussions? - New programs, outreach, town halls? Make sure they are in the CFP list

How do we gauge interest from the specific topics? And will the right people be there to lead the conversation? 

Matteo - way of CFP needs to be rethought on how we do it so that it can be more organized and effective. Ex: diagnostics wg had a session at the Vancouver Colalb Summit. There was a large amt of people from that room. Same for the n-api team. Those were the most successful activities there.

Make sure to add agenda in the CFP so people know how the structure is. 

Sorting out conflicts between the sessions - Joyee did this to help with the summit last time? It was "which session will you go to" 

Individual rooms sessions - do 10 second ice breakers before starting each session

Should have an owner and a facilitator/moderator in each room for each session ie wg chair (owner)

**Comment by WaleedAshraf** [comment]
@ladyleet Yup, Berlin.

**Comment by ladyleet** [comment]
For those interested in joining, @mcollina has updated this to Zoom so we can live broadcast. join here if you are interested in attending. You can also always email me to be added to the invite ladyleet@nodejs.org https://zoom.us/j/122008117

**Comment by ladyleet** [comment]
Also, should out to @WaleedAshraf for brainstorming venues with us!

**Comment by ladyleet** [comment]
Hey @nodejs fam and @the_jsf - we are planning the next Collab Summit in Berlin and really need someone to check out a venue for us this week. 

Can you help? If so just email me ladyleet@nodejs.org #nodejs #jsf

**Comment by hashseed** [comment]
Is there a place to submit talks already? I'd like to give a short talk on building Node with GN.

**Comment by chowdhurian** [comment]
@nodejs/community-committee this week's Summit planning meeting is happening now!

**Comment by keywordnew** [comment]
This week's Summit planning meeting is happening now. On the agenda:

* Eventbrite finalized - to charge or not?
* Swag - Node - ideas?
* Finalize pricing by disucssing with Tracy/Matteo/Myles
* Swag Budget - JSF
* Feedback for CFP submission flow

Feel free to join: https://zoom.us/j/122008117

**Comment by jorydotcom** [comment]
Feedback Requested!!

@keywordnew and I drafted a Google form to collect session submissions and ideas for sessions, and we'd like for you to give it a test drive and let us know what you think. 

This is not 'done'  - it is definitely still a draft. We wanted to give you an idea for how we imagined the flow, and how we could use one form to capture both session submissions and ideas for sessions.

[https://goo.gl/forms/IMljpvzC6sbwqDcz2](https://goo.gl/forms/IMljpvzC6sbwqDcz2)

❤ Jory & Manil

**Comment by mcollina** [comment]
One of the things I like the most of our previous process is how it happened via issues. I think we should retain that, maybe with an issue template? In that way the proposed sessions would still be public.

(+1 on the flow in the google form)

We should also work on the actual text for the call for sessions.

**Comment by mhdawson** [comment]
+1 to using issues for the submissions as well.

**Comment by keywordnew** [comment]
The intent for the form is to:
* help better structure the format of submissions to help organizers curate sessions
* make a publicly viewable list of submissions
* route people to right submission, based on an intent to lead a session or suggest one

This may be better discussed in sync at the next meeting 👍🏽

**Comment by jorydotcom** [comment]
We had a good chat today and great feedback on the CFP form which I wanted to document here for those who missed:
* the summary link has been added to the first page of the form, so people can see what's been submitted so far. We'll also make the responses spreadsheet publicly viewable, and link all of those resources in an issue on GH
* we simplified it by adding the list of ideas to the first page as well, so people don't have to click through to see those (thanks @WaleedAshraf !!!) 
* we are thinking through the voting/interest measurement mechanism to ensure that is public and also that people who have registered to attend the summit are weighing in. 

I've updated the form [here](https://goo.gl/forms/rYjLfIFHVzZr3zzK2) - if there are no objections to move forward with this it would be great to start actively soliciting sessions by the end of the week!

**Comment by keywordnew** [comment]
We've got a [calendar](https://docs.google.com/spreadsheets/d/1r5am7B7eczC5OpkpZsyoU71LHrWS6fUS1gvDFfR1ifw/edit#gid=1481053507) (on the same doc @ladyleet linked earlier!) where we can all put deadlines and delivery dates. It culminates in Summit Day 1 and 2. 🗓

**Comment by refack** [comment]
Could we allocate budget to make this summit is accessible to people with disabilities?

**Comment by mcollina** [comment]
> Could we allocate budget to make this summit is accessible to people with disabilities?

What would you expect? There might be some budget, but we probably need to know what to plan for.

**Comment by Trott** [comment]
> What would you expect? There might be some budget, but we probably need to know what to plan for.

@mcollina In the past, we've had some ability for remote attendees to at least observe/listen. (Not at all Collaborator Summits but for at least one of them.) I would consider that to be the minimum low bar for inclusion of those who cannot physically attend.

If we want to improve on that (and I hope we do), we could find a way to incorporate Zoom attendees as equals (or as equal as possible at least). One thing a few people encourage at my work place that I think works well: Even though most attendees of a meeting are in a room together, everyone should open their laptop and connect via Zoom. People in the room should be sure to *not* connect audio to avoid feedback. But other than that, it puts everyone on an equal footing. Or at least makes remote attendees feel less forgotten. It makes it less of "one box for a room with 50 people, and three other boxes for the remote attendees no one cares about" and more of a "oh, there's 53 boxes on the screen, lots of people, let's all use the chat feature".

I don't think we've ever done much in the way of general accessibility. But that's certainly no reason to at least not discuss what might be possible to improve going forward. I'm not sure what's possible, but if accessibility/inclusivity is a priority for the Board such that we could get the money, and we have one or more motivated volunteers to make it a reality, video streaming of sessions with closed captioning perhaps? Signing for the hearing impaired maybe? (Self-interest note: I have a mild hearing impairment and am a heavy user of captioning when it is available. I do not understand sign language so that one is not for me.)

I'm sure there are lots of other ideas, and I'm sure much of what I wrote above will be out of reach this time around and probably next time around too. But I'm writing it anyway because if we don't talk about it, then we'll certainly *never* get there. :-D

**Comment by mcollina** [comment]
@ladyleet @jorydotcom @WaleedAshraf do you think we could provide some good ambient mic with Foundation budget? In this way we could at least provide a good audio to the remote participants?

**Comment by WaleedAshraf** [comment]
Not sure how much budget we have but we can get a good quilty table/meeting microphone in 50-60€. And as I have experienced, table microphone does improve audio quality as compared to normal laptop microphone.
@mcollina

**Comment by keywordnew** [comment]
Very helpful discussion here about improving accessibility and remote participation. Spinning out the microphone discussion into its own issue.

**Comment by ladyleet** [comment]
We are def looking for more volunteers to help on an ongoing basis! We have a weekly meeting that you can attend!

**Comment by LaRuaNa** [comment]
@ladyleet When is the next meeting? I can help with that :]

**Comment by ladyleet** [comment]
@LaRuaNa if you send me your email address i will add you to the cal invite! ladyleet@nodejs.org

**Comment by LaRuaNa** [comment]
> @LaRuaNa if you send me your email address i will add you to the cal invite! [ladyleet@nodejs.org](mailto:ladyleet@nodejs.org)

Ping @ladyleet  :] Did you receive my email?

**Comment by ladyleet** [comment]
i don't think i did! what is the email so i can look for it?

also, thanks to some new folks who joined the planning meeting this week! 🎉

**Comment by LaRuaNa** [comment]
@ladyleet then doing it just here :]
<img width="120" alt="image" src="https://user-images.githubusercontent.com/2481234/54373790-2b6e8e00-467e-11e9-9ffa-6d9dfd5d0c70.png">

**Comment by ladyleet** [comment]
@LaRuaNa thank you! just sent

**Comment by tniessen** [comment]
I'll try to make it to the summit despite some other things happening on those days :) Let me know if there is anything I can do to help.

**Comment by tniessen** [comment]
Is the date fixed? (Curious because the 30th is a national holiday in Germany.)

**Comment by keywordnew** [comment]
What sort of impact can we expect from the 30th being a national holiday?

**Comment by hashseed** [comment]
> Is the date fixed? (Curious because the 30th is a national holiday in Germany.)

Same thing last year. I don't think this changes anything, unfortunately.

**Comment by tniessen** [comment]
> What sort of impact can we expect from the 30th being a national holiday?

I am not sure, hopefully not much. You can certainly expect [many drunk people in public](https://en.wikipedia.org/wiki/Father%27s_Day#Germany). Do we already have a venue?

> Same thing last year. I don't think this changes anything, unfortunately.

The same holiday was on May 10th last year. I don't think there was a holiday during last year's summit :)

**Comment by keywordnew** [comment]
> Do we already have a venue?

Courtyard Berlin City Centre is ready to go.

**Comment by hashseed** [comment]
> The same holiday was on May 10th last year. I don't think there was a holiday during last year's summit :)

I was thinking of Fronleichnam, which is a public holiday in Bavaria :P

**Comment by ladyleet** [comment]
Just FYI - registration and venue information in issue https://github.com/nodejs/summit/issues/152
